ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -245,8 +245,9 @@ applyShelleyTx cfg wti slot (ShelleyTx _ tx) st0 = do
          wti
          tx
 
-    let st' :: TickedLedgerState (ShelleyBlock proto era) TrackingMK
-        st' = calculateDifference st0
+    let st' :: TickedLedgerState (ShelleyBlock proto era) DiffMK
+        st' = forgetTrackingValues
+            $ calculateDifference st0
```

Comment:
Perhaps we could have a function that calculates the differences directly? I mean `-> DiffMK` instead of `-> TrackingMK` although I suppose this ought to be super cheap already

Reply:
Not important for now I think